test/region_growing.jl

```diff
@@ -0,0 +1,128 @@
+get_wrapped{CT<:Colorant}(p::CT) = Images.accum(typeof(p))(p)
+
+@testset "Seeded Region Growing" begin
+    # 2-D image
+    img = zeros(Gray{N0f8}, 10, 10)
+    img[6:10,4:8] = 0.5
+    img[3:7,2:6] = 0.8
+    seeds = [(CartesianIndex(3,9),1), (CartesianIndex(5,2),2), (CartesianIndex(9,7),3)]
```

Comment:
> the user assigns two different labels to the same point?

The algorithm will run as if only the latter label was assigned to that point.

> the user assigns two different labels to two points in the same connected region of constant value?

The regions will grow independently until the boundary points are encountered, as these points will have same δ value for both regions, they tie out. Hence the boundary points will have label '0'.